### PR TITLE
Freemabd1/quick search update

### PR DESCRIPTION
**Pull request by freemabd1**
removed 
<modifySql dbms="mysql">
<append value"ENGINE=INNODB DEFAULT CHARSET=UTF8"/>
</modifySql>
and added code to quick search

### CODE REVIEW

File: api/db-cdr/changelog-schema/db.changelog-3.xml [hunk 1]
```diff
@@ -8,12 +8,20 @@
...
+      <column name="code"/>
     </createIndex>
+    <createIndex
+            indexName="idx_criteria_name"
+            tableName="criteria"
+            unique="false">
+      <column name="name"/>
+    </createIndex>
+    <modifySql dbms="mysql">
+      <replace replace="INDEX" with="FULLTEXT INDEX"/>
```
Comment (danqrodney) [inline api/db-cdr/changelog-schema/db.changelog-3.xml]: You might want to add a comment saying this applies to both indexes in this change set.

File: api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
     /** TODO: implement dynamic switching of schemas **/
     @Query(value = "select * from cdr.criteria c " +
             "where c.type = :type " +
-            "and match(c.name) against(:value in boolean mode) " +
+            "and (match(c.name) against(:value in boolean mode) or match(c.code) against(:value in boolean mode))" +
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: Can you explain why we want to match against the code as well as the name? Are both visible to the user?

Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: screen shot 2017-11-15 at 2 18 29 pm
So in the above tree, code is bolded and the text on the right is the name database field

Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: So i think the users would find it useful to search by code or something in the name like 'Diabetes'.. which has a bunch of different codes

Comment (johnstill) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: @danqrodney  Think of searching for a book by dewey decimal code or by title.

Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: lol.. well put

Comment (danqrodney) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CriteriaDao.java]: Got it. Thanks!

**Pull request merged**